src/data/usersSampleData.js

```diff
@@ -0,0 +1,4 @@
+export const usersSampleData = {
+    name: "Saige O'Kon",
+    id: 1,
+  };
```

Comment:
Could you add the full user data here? Saige O'Kon should have a pantry array included in their user object! We'll need it for testing the user class.